framework/transport/ieee1905_transport/test/ieee1905_transport_broker_tests.cpp

```diff
@@ -0,0 +1,320 @@
+/* SPDX-License-Identifier: BSD-2-Clause-Patent
+ *
+ * SPDX-FileCopyrightText: 2016-2020 the prplMesh contributors (see AUTHORS.md)
+ *
+ * This code is subject to the terms of the BSD+Patent license.
+ * See LICENSE file for more details.
+ */
+
+#include <gtest/gtest.h>
+
+#include "../ieee1905_transport_broker.h"
+
+#include <sys/uio.h>
+
+#include <bcl/beerocks_os_utils.h>
+#include <bcl/beerocks_thread_base.h>
+
+#include <beerocks/tlvf/beerocks_message.h>
+
+#include <tlvf/CmduMessageTx.h>
+
+#include <mapf/transport/ieee1905_transport_messages.h>
+
+#include <easylogging++.h>
+
+// Initialize easylogging++
+INITIALIZE_EASYLOGGINGPP
+
+namespace beerocks {
+namespace transport {
+namespace broker {
+namespace tests {
+
+using namespace beerocks::transport::messages;
+
+//////////////////////////////////////////////////////////////////////////////
+////////////////////////////// Global Variables //////////////////////////////
+//////////////////////////////////////////////////////////////////////////////
+
+// UDS file for the tests
+static constexpr char broker_uds_file[] = "beerocks_broker_test_uds";
```

Comment:
```suggestion
static const std::string broker_uds_file = "beerocks_broker_test_uds";
```
since c++11 doesn't provide an std::string support for constexpr - it is best to use const std:string over this since you pass `broker_uds_file` to functions/classes that expect std::string which will result with unneeded creation cost every time.

Reply:
It's a test, but OK :)